mkfit-toolfile.spec

```diff
@@ -1,4 +1,4 @@
-### RPM external mkfit-toolfile 2.0.0
+### RPM external mkfit-toolfile 2.0.1
```

Comment:
toolfile version is independent of tool itself. We only change it when the tool definition changes (e.g. add a new libdir, include or lib), So this change is not needed.

Reply:
Thanks, I restored the version.